packages/editor/src/components/editor/Editor.tsx

```diff
@@ -76,13 +78,13 @@ export const Editor = (props: FormEditorProps) => {
             minSize={30}
             className='panel'
             onClick={e => {
-              if (e.target !== e.currentTarget) {
+              if (e.target !== e.currentTarget && !toolbarDiv.current?.contains(e.target as Node)) {
```

Comment:
I had to work with useRef in the toolbar component because the selection with the previous solution became undefined when the toolbar was clicked. This means that the open/close button for the properties did not work correctly.